exporter/model.go

```diff
@@ -30,7 +36,42 @@ type resourceApproximation struct {
 }
 
 type stateApproximation struct {
-	Resources []resourceApproximation `json:"resources"`
+	mutex sync.RWMutex
+	// TODO: use map by type -> should speedup Has function?
+	resources []resourceApproximation
```

Comment:
How big is the slice in general? Iterating through a list can be faster than a map. May want to benchmark/check out generated assembly for insights here.

Reply:
it's proportional to the number of exported objects. In big workspaces it could be easily in range of hundreds of thousands